MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 graft docs
 graft licenses
-graft plasmapy
+graft src
```

Comment:
A benefit of the `src` layout is that it makes it a lot more clear that anything within `src` that is checked into the repo is intended to be part of the distribution.